### ISSUE TITLE
yield oscillator crash in message_channel.py on_pubmsg -- bad array index.

### ISSUE THREAD

**Issue by peertrade** [JoinMarket-Org/joinmarket#595]
Here's a stack trace.

```
  File "yield-generator-oscillator.py", line 715, in main
    irc.run()
  File "/home/joinmarket/joinmarket/joinmarket/irc.py", line 449, in run
    self.__handle_line(line)
  File "/home/joinmarket/joinmarket/joinmarket/irc.py", line 333, in __handle_line
    self.__handle_privmsg(_chunks[0], _chunks[2], get_irc_text(line))
  File "/home/joinmarket/joinmarket/joinmarket/irc.py", line 290, in __handle_privmsg
    self.on_pubmsg(nick, message)
  File "/home/joinmarket/joinmarket/joinmarket/message_channel.py", line 241, in on_pubmsg
    if message[0] != COMMAND_PREFIX:
IndexError: string index out of range
```

**Comment by AdamISZ** [comment]
Fixed in 69cb93c8112a76ea53b672e15d280660ba5d0dd1

**Comment by peertrade** [comment]
quick work, thanks!
